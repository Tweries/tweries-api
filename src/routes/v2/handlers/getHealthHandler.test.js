const {
  matcherV2,
  mockSend,
  reqBase,
  resBase
} = require('../../handlers/mocks');
const getHealthHandler = require('./getHealthHandler');

test('getHealthHandler', () => {
  const req = { ...reqBase };
  const res = { ...resBase };

  getHealthHandler(req, res);

  expect(mockSend.mock.calls[0][0]).toMatchSnapshot(matcherV2);
});
