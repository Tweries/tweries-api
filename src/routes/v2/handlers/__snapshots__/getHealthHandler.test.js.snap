// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getHealthHandler 1`] = `
Object {
  "data": Object {
    "message": "ðŸ‘‹ hello, I'm healty",
  },
  "error": undefined,
  "meta": Object {
    "env": "test",
    "name": "tweries-api",
    "timestamp": Any<Number>,
    "version": Any<String>,
  },
}
`;
